src/i18n/TextCollection/i18nTextCollector.php

```diff
@@ -396,11 +399,11 @@ protected function getEntitiesByModule()
     {
         // A master string tables array (one mst per module)
         $entitiesByModule = [];
-        $modules = ModuleLoader::inst()->getManifest()->getModules();
-        foreach ($modules as $module) {
+        $modules = $this->getModulesAndThemes();
+        foreach ($modules as $moduleName => $module) {
             // we store the master string tables
             $processedEntities = $this->processModule($module);
-            $moduleName = $module->getName();
+            $moduleName = str_starts_with($moduleName, THEMES_DIR) ? $moduleName : $module->getName();
```

Comment:
Sorry, didn't notice this before. We can't use `str_starts_with` as that is PHP 8 only, we still need to support PHP 7.4 in 4.x framework, so please use `strpos`.

Reply:
done